dietpi/dietpi-login

```diff
@@ -185,7 +185,7 @@ Please login again as user "root" with password "dietpi", respectively the one y
 			if (( $G_DIETPI_INSTALL_STAGE == 0 )); then
 
 				# Start DietPi-Update
-				/boot/dietpi/dietpi-update 1 # Sets /boot/dietpi/.install_stage > G_DIETPI_INSTALL_STAGE=1
+				/boot/dietpi/dietpi-update 1 2>&1 | tee -a "$FP_DIETPI_FIRSTRUNSETUP_LOG" # Sets /boot/dietpi/.install_stage > G_DIETPI_INSTALL_STAGE=1
```

Comment:
```suggestion
				/boot/dietpi/dietpi-update 1 2>&1 | tee "$FP_DIETPI_FIRSTRUNSETUP_LOG" # Sets /boot/dietpi/.install_stage > G_DIETPI_INSTALL_STAGE=1
```
The first one can remain clearing the file. So when you need to rerun for some reason only the relevant last iteration is part of the log, making it easier to see what the final issue was (if any). On the other hand, when something in APT/dpkg fails which causes follow up issues, previous iterations can be interesting as well. For me either is okay, clearing the file once per loop would just be the smaller change.

Reply:
Oops i didn't even see you can just commit the suggested change.
Regardless suggestion applied via force push.